### PR TITLE
Added Ethereum DApps in Motion

### PR DESCRIPTION
**Pull request by roziunicorn**
Hello,
would you please add this video?

Thanks

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -91,6 +92,9 @@ Easily access this list @ [`dapps.cool`](http://dapps.cool).
 
 - [Ethereum Jobs](http://jobs.ethercasts.com/) - The premier job board and developer community space specifically for Ethereum by [EtherCasts](https://github.com/EtherCasts)
 
+## Videos
+- [Ethereum DApps in Motion](https://www.manning.com/livevideo/ethereum-dapps-in-motion) Julien Klepatch teaches you how to build DApps using the Ethereum toolchain and an emerging body of best practices.
```
Comment (jasonwalsh) [inline readme.md]: Remove the trailing whitespace.

File: readme.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ Easily access this list @ [`dapps.cool`](http://dapps.cool).
   - [Entertainment](#entertainment)
 - [Developer Tools](#developer-tools)
 - [Jobs](#jobs)
+- [Videos](#Videos)
```
Comment (jasonwalsh) [inline readme.md:23]: Change this line:

```diff
- [Videos](#Videos)
+ [Videos](#videos)
```

**Review by jasonwalsh — CHANGES_REQUESTED** [review]
Hey @roziunicorn just some minor changes and then I will merge. Thanks!

**Comment by roziunicorn** [comment]
Thank you so much:) and Happy New Year!

**Comment by jasonwalsh** [comment]
@roziunicorn this isn't resolved. You still need to make the changes I requested for me to accept the PR.

**Comment by roziunicorn** [comment]
Sorry, just did it!

**Pull request closed without merging**